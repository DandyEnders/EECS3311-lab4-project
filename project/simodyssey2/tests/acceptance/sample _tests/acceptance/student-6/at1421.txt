move(E)
abort
move(E)
move(SE)
move(S)
move(NE)
move(E)
move(SW)
move(NE)
pass
wormhole
abort
move(NW)
move(SE)
move(N)
move(NE)
liftoff
move(NW)
move(SE)
land
move(N)
test(66,80,80,80,94)
abort
move(E)
move(SE)
land
move(E)
move(W)
land
move(E)
move(N)
