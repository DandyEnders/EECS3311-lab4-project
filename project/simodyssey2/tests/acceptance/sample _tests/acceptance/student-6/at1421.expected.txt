  state:0.0, ok
  Welcome! Try test(3,5,7,15,30)
->move(E)
  state:0.1, error
  Negative on that request:no mission in progress.
->abort
  state:0.2, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.3, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.4, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.5, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.6, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.7, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.8, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.9, error
  Negative on that request:no mission in progress.
->pass
  state:0.10, error
  Negative on that request:no mission in progress.
->wormhole
  state:0.11, error
  Negative on that request:no mission in progress.
->abort
  state:0.12, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.13, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.14, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.15, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.16, error
  Negative on that request:no mission in progress.
->liftoff
  state:0.17, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.18, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.19, error
  Negative on that request:no mission in progress.
->land
  state:0.20, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.21, error
  Negative on that request:no mission in progress.
->test(66,80,80,80,94)
  state:1.0, mode:test, ok
  Movement:none
  Sectors:
    [1,1]->[0,E],[1,A],-,-
    [1,2]->[2,P],[3,A],[4,P],[-8,Y]
    [1,3]->[5,A],-,-,-
    [1,4]->[6,A],[7,A],[8,P],-
    [1,5]->[9,A],[-11,*],-,-
    [2,1]->[10,A],-,-,-
    [2,2]->[11,A],[12,A],[13,A],[-6,Y]
    [2,3]->[14,P],-,-,-
    [2,4]->[15,A],[16,P],-,-
    [2,5]->[17,A],[18,A],[-4,*],-
    [3,1]->[19,P],[20,J],[21,A],-
    [3,2]->[22,A],[23,A],[24,A],-
    [3,3]->[-1,O],-,-,-
    [3,4]->[25,A],[26,A],[27,A],[-3,W]
    [3,5]->[28,A],[-7,W],-,-
    [4,1]->[29,P],-,-,-
    [4,2]->[30,A],-,-,-
    [4,3]->[31,P],-,-,-
    [4,4]->[32,P],[-10,W],-,-
    [4,5]->[33,A],-,-,-
    [5,1]->[34,A],[35,A],-,-
    [5,2]->[36,A],[-5,Y],-,-
    [5,3]->[37,A],[38,A],[-2,W],-
    [5,4]->[39,A],[-9,Y],-,-
    [5,5]->[40,A],[41,J],-,-
  Descriptions:
    [-11,*]->Luminosity:5
    [-10,W]->
    [-9,Y]->Luminosity:2
    [-8,Y]->Luminosity:2
    [-7,W]->
    [-6,Y]->Luminosity:2
    [-5,Y]->Luminosity:2
    [-4,*]->Luminosity:5
    [-3,W]->
    [-2,W]->
    [-1,O]->
    [0,E]->fuel:3/3, life:3/3, landed?:F
    [1,A]->turns_left:1
    [2,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [3,A]->turns_left:2
    [4,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [5,A]->turns_left:0
    [6,A]->turns_left:1
    [7,A]->turns_left:0
    [8,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [9,A]->turns_left:1
    [10,A]->turns_left:0
    [11,A]->turns_left:1
    [12,A]->turns_left:0
    [13,A]->turns_left:1
    [14,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [15,A]->turns_left:0
    [16,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [17,A]->turns_left:0
    [18,A]->turns_left:1
    [19,P]->attached?:F, support_life?:F, visited?:F, turns_left:1
    [20,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
    [21,A]->turns_left:0
    [22,A]->turns_left:1
    [23,A]->turns_left:0
    [24,A]->turns_left:0
    [25,A]->turns_left:1
    [26,A]->turns_left:1
    [27,A]->turns_left:1
    [28,A]->turns_left:0
    [29,P]->attached?:F, support_life?:F, visited?:F, turns_left:0
    [30,A]->turns_left:2
    [31,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [32,P]->attached?:F, support_life?:F, visited?:F, turns_left:2
    [33,A]->turns_left:2
    [34,A]->turns_left:1
    [35,A]->turns_left:1
    [36,A]->turns_left:2
    [37,A]->turns_left:2
    [38,A]->turns_left:2
    [39,A]->turns_left:0
    [40,A]->turns_left:2
    [41,J]->fuel:5/5, load:0/2, actions_left_until_reproduction:2/2, turns_left:1
  Deaths This Turn:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EA--   PAPY   A---   AAP-   A*--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    A---   AAAY   P---   AP--   AA*-   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PJA-   AAA-   O---   AAAW   AW--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    P---   A---   P---   PW--   A---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    AA--   AY--   AAW-   AY--   AJ--   
->abort
  state:1.1, ok
  Mission aborted. Try test(3,5,7,15,30)
->move(E)
  state:1.2, error
  Negative on that request:no mission in progress.
->move(SE)
  state:1.3, error
  Negative on that request:no mission in progress.
->land
  state:1.4, error
  Negative on that request:no mission in progress.
->move(E)
  state:1.5, error
  Negative on that request:no mission in progress.
->move(W)
  state:1.6, error
  Negative on that request:no mission in progress.
->land
  state:1.7, error
  Negative on that request:no mission in progress.
->move(E)
  state:1.8, error
  Negative on that request:no mission in progress.
->move(N)
  state:1.9, error
  Negative on that request:no mission in progress.
